src/workerd/api/http.c++

```diff
@@ -1461,7 +1461,10 @@ jsg::Ref<Response> Response::constructor(jsg::Lock& js,
       }
     }
   } else {
-    statusText = kj::str(defaultStatusText(statusCode));
+    // A response has an associated status message. Unless stated otherwise it is the empty byte sequence.
+    // https://fetch.spec.whatwg.org/#concept-response-status-message
+
+    statusText.emplace(kj::str(""));
```

Comment:
Hmm.. Unfortunately -- and this makes me sad -- there's a small possibility that this could be a breaking change for someone. I think we likely need a compat flag for this.